### PR TITLE
Update headless parameter in "Use WebDriver to automate Microsoft Edge"

### PR DESCRIPTION
**Pull request by Delta456**
Rendered article for review:
* **Use WebDriver to automate Microsoft Edge**
   * Internal preview: https://review.learn.microsoft.com/microsoft-edge/webdriver-chromium/?branch=pr-en-us-3323#pass-extra-command-line-arguments
   * Public preview: https://github.com/Delta456/edge-developer/blob/headless/microsoft-edge/webdriver-chromium/index.md#pass-extra-command-line-arguments
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/webdriver-chromium/#pass-extra-command-line-arguments)
   * The current docs have `headless` as a parameter to run the headless mode which is outdated and doesn't work:
      * `options.AddArgument("headless");`
      * `options.add_argument("headless")`
      * `options.addArguments("headless");`
      * `options.addArguments("headless");`
   * `--headless=new` is the way to run in headless mode.  Updated the code in the C#, Python, Java, and JavaScript tabs, to:
      * `options.AddArgument("--headless=new");`
      * `options.add_argument("--headless=new")`
      * `options.addArguments("--headless=new");`
      * `options.addArguments("--headless=new");`

Ref: https://www.selenium.dev/blog/2023/headless-is-going-away/

AB#55262126

### CODE REVIEW

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer/Editor.  
I added lead-ins above two code snippets.  I didn't confirm the code for each platform.  Instead of Brandon, Gavin Xu is a backup tech reviewer.

File: microsoft-edge/webdriver-chromium/index.md [hunk 1]
```diff
@@ -366,7 +366,7 @@ EdgeDriver driver = new EdgeDriver(options);
 const edge = require('selenium-webdriver/edge');
 
 let options = new edge.Options();
-options.addArguments("headless");
+options.addArguments("--headless=new");
```
Comment (mikehoffms) [inline microsoft-edge/webdriver-chromium/index.md:369]: https://www.selenium.dev/blog/2023/headless-is-going-away/#how-can-i-set-headless-mode-from-now-on
shows different code, for JS only:
image

Comment (Delta456) [reply microsoft-edge/webdriver-chromium/index.md:369]: https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples%2Fjavascript%2Ftest%2Fbrowser%2FedgeSpecificCaps.spec.js#L12

Indeed it is.

Do you want me to change this?

Comment (guangyuexu) [reply microsoft-edge/webdriver-chromium/index.md:369]: No need to change, the current JS codes to apply `--headless=new` worked.

**Review by guangyuexu — APPROVED** [review]
LGTM

File: microsoft-edge/webdriver-chromium/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webdriver-chromium/index.md:369]: closing.  thx for testing @guangyuexu

**Pull request merged**